app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,47 @@
+class QuestionsController < ApplicationController
+
+	def index
+			if current_user.active?
+				@question = Question.all 
```

Comment:
Sort questions by latest create date.
Use `@questions` since we have multiple questions stored in that instance variable

Reply:
okiee.. will work on that..